presentation/components/src/presentation-components/tree/controlled/TreeHooks.ts

```diff
@@ -155,12 +290,11 @@ function createDataProvider(props: PresentationTreeNodeLoaderProps): IPresentati
   if (props.dataProvider) {
     dataProvider = props.dataProvider;
   } else {
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    const { preloadingEnabled, dataProvider: testDataProvider, ...providerProps } = props;
+    const { preloadingEnabled: _preloadingEnabled, dataProvider: _dataProvider, ...providerProps } = props;
     dataProvider = new PresentationTreeDataProvider(providerProps);
   }
   if (props.preloadingEnabled && dataProvider.loadHierarchy) {
-    dataProvider.loadHierarchy(); // eslint-disable-line @typescript-eslint/no-floating-promises
+    void dataProvider.loadHierarchy();

```

Comment:
what's the point of this change?

Reply:
I removed the `eslint-disable` comment and this was the suggested fix.

Reply:
I think the `eslint-disable` comment in this case is very useful - it tells people who look at the code that `loadHierarchy` was called without `await` on purpose.